chia/full_node/full_node.py

```diff
@@ -350,9 +357,9 @@ async def manage(self) -> AsyncIterator[None]:
             self.initialized = True
 
             try:
-                async with contextlib.AsyncExitStack() as exit_stack:
+                async with contextlib.AsyncExitStack() as aexit_stack:
```

Comment:
The local `mypy` caught this subtle problem (`exit_stack` is already typed in an incompatible way in this scope).